hack/util.sh

```diff
@@ -660,7 +660,8 @@ function install_registry() {
 readonly -f install_registry
 
 function wait_for_registry() {
```

Comment:
**TODO**: put a comment that all the registry re-deployments wipe out registry storage.